### DISCUSSION TITLE
Welcome

### DISCUSSION THREAD

**Discussion by Cwejman** [Hjulverkstan/hjulverkstan#14] [Announcements]
Hi!

Most of you have been invited, while you have not accepted the invitation yet I'm curious if you'll find this message.

On our demo a few weeks ago we discussed using slack as a coms platform however I thought we could try discussions here on GitHub first.

I've just downloaded the GitHub app on my phone, not as strait forward as Telegram for communications but would be nice to get notifications. Let's try it out and see how it works.

Bear in mind that while we are still in private mode for the repo the main branch is not protected as this is a paid feature. So be cautious and don't push to main directly...

If you see this message please give a comment and I will know that it found you.

Cheers,
Jona

**Comment by Cwejman** [comment]
@Hjulverkstan/devs Hi, I thought I'd try to ping the team as I've just created one. Maybe that is an easy way to ping everyone.

**Comment by Cwejman** [comment]
@Hjulverkstan/devs Hi everyone. Are you getting notifications yet? :) I have now update the backlog with the most of everything we have thought of. Probably some tasks missing that where not existent in Jira that we used before for backlog tracking. You are welcome to take a look at the backlog, and grab a low hanging fruit if you feel like it or just get accustomed to what is lying up ahead.

Cheers

**Reply by SamuelSiesjo** [reply]
I'm recieving them. I will take a look as soon as i have the chance.

**Reply by GabMeloche** [reply]
Receiving them as well

**Comment by daniel-svendsen** [comment]
Recieving aswell 👍🙂

Skickat från Outlook för Android<https://aka.ms/AAb9ysg>
________________________________
